includes/breadcrumbs.inc

```diff
@@ -114,6 +113,7 @@ function islandora_solr_get_breadcrumb_parent($pid) {
     'facet' => 'false',
   );
   $solr_build->buildQuery($solr_query, $solr_params);
+  $solr_build->solrStart = 0;
```

Comment:
ohh.. this is all because we are using the external params to build the query? good catch! So, in how many other creative ways this could happen for other arguments? Maybe breadcrumbs could totally disregard arguments when dealing with the query creation? Or is Start only concern?

Reply:
Having looked at replacing ApacheSolrClient all I can say is our IslandoraSolrQueryProcessor is designed so anyone can reach into its guts and play around. This is helpful for adding namespace and user restrictions, but can hurt as in this case.

For this case, the issue really is that the query returns `1` item (the parent) but on page 2 (which is really `page=1`) the query_processer sets `start` to the limit (default is 48 I think) * page. So you `start=48` and get nothing back.

To change it would require a huge process of making it cleaner (getters/setters) and then altering all the community code, and that doesn't take into account all the "in the wild" uses. 🤷‍♂️

Reply:
Your observation matches my Friday's observation of wild roaming code. Approved and waiting for 24 hours mark. This is a bug fix so will make it into the release even after code freeze